modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -59,9 +59,13 @@ Run performance and load testing on your Mule applications to determine the numb
 
 == Internal Load Balancer
 
-Inbound traffic is processed using an internal load balancer managed by Anypoint Runtime Fabric. Since this load balancer is responsible for TLS termination, the amount of resources required scales based on the number of incoming connections and the average payload size for each request.
+Inbound traffic is processed using an internal load balancer managed by Anypoint Runtime Fabric. Since this load balancer is responsible for TLS termination, the number of resources required scales based on the number of incoming connections and the average payload size for each request.
 
-The following table lists the approximate requests averaging 10 KB that can be served with a single replica of the internal load balancer based on the amount of CPU cores.
+Performance tests were run on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. The load generator that was used was hosted on a separate AWS instance in the same region. The M4 instances featured a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2, which runs at a base clock rate of 2.4 GHz. Using Intel Turbo Boost, the clock rate can go as high as 3.0.
+
+A C++ based load generator with better efficiency in SSL connections was used to yield the maximum throughput.
+
+The following table lists the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer based on the amount of CPU cores.
```

Comment:
"Performance tests were run..." is bothering me but I am not sure how to address it. I know it is unimportant who or what ran the tests (active voice), but I still want to find an active way to phrase this. Can we use something like, "In performance tests run on an Anypoint Runtime Fabric cluster with..." or "Based on performance tests run..."?

Is your audience familiar with all of the abbreviations used here?

amount > number

"CPU cores." > "CPU cores:"    (colon)